src/pages/user/index.tsx

```diff
@@ -169,6 +36,18 @@ const Account: React.FunctionComponent<
     if (interval === 'month' && interval_count === 1) return 'month'
     if (interval === 'year' && interval_count === 1) return 'year'
   }
+
+  React.useEffect(() => {
+    const loadAccountForSlug = async (slug) => {
+      if (slug) {
+        const account: any = await loadAccount(slug, authToken)
+        setAccount(account)
+      }
+    }
+
+    loadAccountForSlug(slug)
+  }, [slug, authToken])
```

Comment:
Get the slug from `useViewer()` above and then `loadAccount` with that. Once the account is loaded, if there is a `stripe_customer_id`, then the other useEffect will be able to look up that info.